10c-spatial-transcriptomics.Rmd

```diff
@@ -1,98 +1,158 @@
-1. **Generate Hypotheses and Identify Biomarkers:** Spatial transcriptomic analysis can generate hypotheses and identify potential biomarkers associated with specific tissue functions or disease states. By examining gene expression patterns in the spatial context, researchers can discover spatially restricted gene signatures and potential diagnostic or prognostic markers. This knowledge has implications for disease research, precision medicine, and therapeutic targeting.
+1.  **Describe tissue-specific cellular neighborhoods of cell types and cell type sub-populations:** Although scRNA-seq continues to be a powerful method to assign biological identities to a mixture of cells, integrated analysis of ST combined with scRNA-seq adds crucial information to cell phenotypes by describing the neighborhoods where cells occur [@longo2021integrating]. Many methods to phenotype ST data are available, with most of them relying on the availability of a curated (scRNA-seq) cell type reference. Once cell identities have been determined, clustering or spatial statistics can be applied to describe the composition of tissue niches or domains. The explosion of ST data has resulted on novel and comprehensive tissue- or disease-specific atlases, not only describing the cell types within organs, but also the functional cell-cell relationships that result from spatial organization (e.g., @guilliams2022spatial; @wu2021single).
+2.  **Uncover spatially regulated biological processes:** With ST data, there comes the ability to detect genes or gene pathways that are expressed in specific areas within tissues (i.e., spatially-restricted expression). Detecting genes with spatially-restricted expression is key to achieve further understanding of specific biological processes, such as tissue gradients, cell differentiation, or signaling pathways. For example, cancer researchers are now able to study signaling pathways restricted to the tumor-stroma interface [@hunter2021spatially], which could lead to the discovery of mechanisms representing cancer vulnerabilities resulting from interactions between the tumor and stroma cells.
+3.  **Investigate cell-cell interactions:** From basic to applied tissue biology research, the study of cell-cell interactions is of high interest, especially the interactions that occur via ligand-receptor pairs. The construction of comprehensive databases of ligand-receptor interactions has been possible due the large amounts of single-cell data sets produced by researchers. A major contribution of ST to the study of tissue biology is the addition of the spatial context to previously identified ligand-receptor interactions. Because single-cell RNA-seq requires physical separation of cells, current ligand-receptor databases represent hypotheses which ST can help to address by using models of spatial co-localization, enabling in-situ examination of cell-cell interactions and communication [@REFERENCES].
```

Comment:
```suggestion
3.  **Investigate cell-cell interactions:** From basic to applied tissue biology research, the study of cell-cell interactions is of high interest, especially the interactions that occur via ligand-receptor pairs. The construction of comprehensive databases of ligand-receptor interactions has been possible due the large amounts of single-cell data sets produced by researchers. A major contribution of ST to the study of tissue biology is the addition of the spatial context to previously identified ligand-receptor interactions. Because single-cell RNA-seq requires physical separation of cells, current ligand-receptor databases represent hypotheses which ST can help to address by using models of spatial co-localization, enabling in-situ examination of cell-cell interactions and communication [@raredon2023comprehensive; wang2023promising].
```